StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/Models/FinancialConnectionsGenericInfoScreen.swift

```diff
@@ -21,10 +21,11 @@ struct FinancialConnectionsGenericInfoScreen: Decodable {
         let icon: FinancialConnectionsImage?
         let alignment: Alignment?
 
-        enum Alignment: String, Decodable {
+        enum Alignment: String, SafeEnumCodable {
```

Comment:
These could just be `SafeEnumDecodable`, right? This is also fine though 👍

Reply:
ahh true, I just copied the enums in the `FinancialConnectionsSheet` so I'll just go with the precedent haha

https://github.com/stripe/stripe-ios/blob/6968678b/StripeFinancialConnections/StripeFinancialConnections/Source/API%20Bindings/Models/FinancialConnectionsSession.swift#L72-L78